### PR TITLE
[docs] add documentation for usage

### PR DESCRIPTION
**Pull request by zach-bob**
This is the first version of user documentation. As of pushing this PR, we have a live version deployed to fly.io that is ready for using and feedback!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,28 +3,36 @@
...
+5. Lets go ahead and configure webhooks for your repository. In github, you're going to need to be a repository admin. If you are not an admin, have an admin complete this step. From your repository, navigate to `Settings > Webhooks > Add webhook`. From here, enter in the endpoint: `https://captains-log-dev.fly.dev/webhook`, then click `Let me select individual events`. Scroll down, unselect `Pushes` and select `Releases`. Then scroll down and click the `Add webhook` button. If there are multiple projects with different repositories, then this step will need to be applied to each repository.
```
Comment (smdavila) [inline README.md:22]: is this right? the dev version?

File: README.md [hunk 2]
```diff
@@ -3,28 +3,36 @@
...
+6. That's it! Your Slack channel should get release notifications as soon as they are released in github. If this is not the case, you may reach out to `@zachbob` on Slack for some assistance.
+7. Once your repository is configured, all release notes and release events are posted in this [Confluence Directory](https://mojotech.atlassian.net/wiki/spaces/PM/pages/29458448/The+Captain).
```
Comment (smdavila) [inline README.md:24]: didn't we discuss creating the releases under Product Management > Client Name > Project Name? is that still an option for MVP? (maybe even hardcoding for Source Advisors to test?)

File: README.md [hunk 3]
```diff
@@ -3,28 +3,36 @@
...
+8. If you wish to manually create release notes, this can be done by entering this `/captainslog` command and filling out the modal form.
```
Comment (smdavila) [inline README.md:25]: maybe add some phrasing to let users know this is not connected to a github release and will simply post whatever entered in the modal to confluence under the same configured parent page.

File: README.md (hunk 1 above)
Comment (zach-bob) [reply README.md:22]: Yes, that `https://captains-log-dev.fly.dev/webhook` is the fly.io deployed url. I definitely goofed when making this. I was thinking of having asking for a CNAME alias for mojotech.com of captainslog.mojotech.com

File: README.md (hunk 2 above)
Comment (zach-bob) [reply README.md:24]: I was trying to look into persisting to folders in [this ticket](https://mojotech.atlassian.net/jira/software/projects/MRNN/boards/22?selectedIssue=MRNN-27), but this seems like a whole new beast right now. We may have to spin up a SPIKE for this

**Review by smdavila — APPROVED** [review]

**Pull request merged**